crates/host/Cargo.toml

```diff
@@ -7,8 +7,8 @@ authors = ["thedavidmeister", "thedavidmeister@gmail.com"]
 edition = "2021"
 
 [dependencies]
-wasmer = { version = "=4.3.6", optional = true, default-feature = false }
-wasmer-middlewares = { version = "=4.3.6", optional = true, default-feature = false }
+wasmer = { version = "=4.3.6", optional = true, default-features = false }
+wasmer-middlewares = { version = "=4.3.6", optional = true, default-features = false }
```

Comment:
This took me way too long to find. This is a horrible little mistake to catch!

Reply:
Why does cargo check not catch that?

Reply:
Good question! Just put this back and ran `cargo check`. Doesn't find it.

I guess Cargo is ignoring unknown fields? Which is something I hate with a passion. If people want custom properties, make them use `x-property-name` and reject all other inputs. We've known this is a good way to design software for 50 years because you can find it in specs from the 70s... /endrant :)

Reply:
oof my bad! :grimacing:

Reply:
All good, I blame Cargo, not the typo :)